### DISCUSSION TITLE
Sidebar to organize all shared notes

### DISCUSSION THREAD

**Discussion by AbdelrhmanUZaki** [alangrainger/share-note#56] [Vote for upcoming features]
the home page would be better if it has a side bar like normal websites to make it easier to view all files

**Comment by alangrainger** [comment]
Hi @AbdelrhmanUZaki - do you mean that on a shared note there would be a left or right sidebar that shows all your shared notes?

That sounds like an awful idea for security. 😬 

If not, what do you mean?

**Reply by AbdelrhmanUZaki** [reply]
Hi @alangrainger - no not all shared notes, but notes I decide to collect together
I spent 3 days to find a tool that did with as I need but I didn't found
Your tool is awesome -honestly the easiest way I found- but two main things we still need
first is that collecting some of notes in structured way as in vault, and the other thing is the ability to share all notes at once, not one by one

**Comment by alangrainger** [comment]
This can already be accomplished with Share Note by creating a sidebar inside a note with CSS, with links to your other notes. The note will then be shared with that sidebar.

Otherwise to have a full multi-note experience, you should use one of the vault publishing plugins, rather than this note sharing plugin.

I will close this request as anything else is outside scope for this plugin.
